drivers/Cargo.toml

```diff
@@ -16,12 +16,14 @@ caliptra-image-types.workspace = true
 caliptra-lms-types.workspace = true
 caliptra-registers.workspace = true
 cfg-if.workspace = true
+dpe = { workspace = true, optional = true }
```

Comment:
Discussed offline: a future PR will move the DpeInstance struct to a new dpe-types crate, and caliptra-drivers will depend on that instead of all of dpe.